fstatat(2) path argument is allowed to be NULL
with AT_EMPTY_PATH since Linux 6.11

--- a/src/stat/fstatat.c
+++ b/src/stat/fstatat.c
@@ -78,7 +78,7 @@
 	int ret;
 	struct kstat kst;
 
-	if (flag==AT_EMPTY_PATH && fd>=0 && !*path) {
+	if (flag==AT_EMPTY_PATH && fd>=0 && (!path || !*path)) {
 		ret = __syscall(SYS_fstat, fd, &kst);
 		if (ret==-EBADF && __syscall(SYS_fcntl, fd, F_GETFD)>=0) {
 			ret = __syscall(SYS_fstatat, fd, path, &kst, flag);
